### ISSUE TITLE
Synoptic Panel

### ISSUE THREAD

**Issue by bryancoker** [okviz/free-visuals#168]
Issue with visual not allowing me to connect to the saved .svg file that i've created.  I add the visual to the canvas, but the option to browse for the SVG file does not appear. I've attached a copy of the power BI report for repro purpuses.  However, most simply, i've attached an image of what i understand should show up for me after i download the custom visual and drag it to my canvas. Once i select some data for the visual, i'm expecting to see the option shown in the image file to show up for me but it never does.  THerefore i cannot link the SVG file to my visual.  Am i missing a step?
![issue](https://user-images.githubusercontent.com/19196534/87687900-240c3880-c73b-11ea-8652-0cf38310be52.png)

**Comment by bryancoker** [comment]
Hi Daniel, I provided a report via email.  Did that not come through?  I tried adding directly to this thread, but it wouldn't let me attach a pix.  is there another option? What screenshot would be helpful?  I was expecting to see a page that i showed above, but instead, i get a blank visual on my canvas.  My visual shows the title of what ever data i select, but it does not give me the option to "select map".
